ability_item_usage_zuus.lua

```diff
@@ -19,9 +19,13 @@ end
 local Talents = {}
 local Abilities = {}
 local AbilitiesReal = {}
+npcBot.ult = {}

```

Comment:
Could you add some comments to that variable?

Reply:
done